Makefile

```diff
@@ -50,10 +50,8 @@ install: reptyr
 	install -m 644 reptyr.1 $(DESTDIR)$(MANDIR)/man1/reptyr.1
 	install -d -m 755 $(DESTDIR)$(MANDIR)/fr/man1
 	install -m 644 reptyr.fr.1 $(DESTDIR)$(MANDIR)/fr/man1/reptyr.1
-	bashcompdir=$(BASHCOMPDIR) ; \
-	test -z "$$bashcompdir" && bashcompdir=/etc/bash_completion.d ; \
-	install -d -m 755 $(DESTDIR)$$bashcompdir ; \
-	install -m 644 reptyr.bash $(DESTDIR)$$bashcompdir/reptyr
+	install -d -m 755 $(DESTDIR)$(BASHCOMPDIR) ; \
+	install -m 644 reptyr.bash $(DESTDIR)$(BASHCOMPDIR)/reptyr
```

Comment:
Where does `$(DESTDIR)` come from? I know that `autoconf` works with `PREFIX` and `DESTDIR`, but I didn't see an `autoconf` setup in your build system, just the `Makefile`. What happens if someone sets the `DESTDIR` to e.g. `foobar`? Does line 54 expand to `install -m 644 reptyr.bash foobar/usr/local/share/bash-completions/reptyr`? That seems a little odd to me.

Reply:
Make allows referencing undefined variables, and they'll just expand to an empty string. I just cargo-culted the existing pattern we have, which seems to match [the convention recommended by GNU Make](https://www.gnu.org/software/make/manual/make.html#DESTDIR).